### PR TITLE
[dev-dev-only-tab] Developer-only tabs

### PR DESCRIPTION
**Pull request by Javran**
as described in #1270 

- submenu "Dev-Only" in strategy room. only visible when `SettingsDevOnlyPages` is turned on.
- a playground tab for developers to do experiments, see details in https://github.com/KC3Kai/KC3Kai/blob/dev-dev-only-tab/src/pages/strategy/tabs/playground/README.md

### CODE REVIEW

File: src/pages/strategy/tabs/playground/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+This Strategy Room Tab is used as a sandbox for helping you with KC3 development.
+
+Not that all changes to this directory should not be commited to the repository,
+you can do the following so Git will ignore all changes you made to this directory:
+
+```bash
+$ cd </path/to/repo/home>
+$ cd src/pages/strategy/tabs/playground
+$ git update-index --assume-unchanged *
```
Comment (dragonjet) [inline src/pages/strategy/tabs/playground/README.md]: nice I use this too in other repos.

additional, if one is using tortoisegit, they can right-click the file in the commit dialog, then choose "assume unchanged"

File: src/library/modules/Translation.js [hunk 2]
```diff
@@ -47,16 +47,39 @@
 			// Apply HTML language code
 			$("html").attr("lang", ConfigManager.language);
 		},
+
+		/*
+		  Recursively changing any non-object value "v" into "{val: v, tag: <tag>}".
+		 */
```
Comment (dragonjet) [inline src/library/modules/Translation.js]: won't this add tag only to the deepest objects?

**Comment by LostLogia4** [comment]
how do i enable it?

File: src/library/modules/Translation.js (hunk 2 above)
Comment (Javran) [inline src/library/modules/Translation.js]: @dragonjet yes, only deepest non-object values.

**Comment by Javran** [comment]
@LostLogia4 In Global Settings  check `SettingsDevOnlyPages` (for now it has not yet been translated)

**Pull request merged**